monitor/external_export/graphana.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+import json
+import os
+import requests
+import string
+import argparse
+
+from monitor import get_lambda_names
+
+
+mon_home = os.getenv('DSS_MON_HOME')
+
+def load_template_file(template_path: str):
+    """Loads JSON template to dict."""
+    with open(template_path) as file:
+        data = json.load(file)
+    return data
+
+def format_panel_positioning(dashboard_src:dict, dashboard_dst: dict):
+    '''Use the dashboard_src to position panels correctly on the dashboard_dest'''
+    for panel in dashboard_dst['panels']:
+        for source_panel in dashboard_src.get("panels"):
+            if panel['title'] == source_panel['title']:
+                panel["gridPos"] = source_panel['gridPos']
+                break
+            print(f"Unable to locate source panel with title: {panel['title']}")
```

Comment:
This will print if the first one found is not the correct source panel.  Should this be dedented?

Reply:
shall fix